### PR TITLE
add desc of required instance tag

### PR DESCRIPTION
**Pull request by geoffcline**
**1. Issue, if available:**
#432 

**2. Description of changes:**
add new section to launch template guide; section describes a requirement that instances be tagged with `Kubernetes.io [...] : owned`; clarify that the IAM instance profile is not optional; 

**3. Does this change impact docs?**
- [x] Yes, PR includes docs updates
- [ ] Yes, issue opened: *link to issue*
- [ ] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/docs/AWS/launch-templates.md [hunk 1]
```diff
@@ -41,6 +41,20 @@ AMI (Amazon Machine Image), is the base image/VM for a launch template.
...
+
+The launch template must include an "instance profile" -- an IAM role.
+
+The instance profile must include *at least* the permissions of the default Karpenter node instance profile. See the default role, `KarpenterNodeRole`, in the full example below for more information.
+
+See also, [the managed policy "AmazonEKSWorkerNodePolicy"](https://docs.aws.amazon.com/eks/latest/userguide/security-iam-awsmanpol.html#security-iam-awsmanpol-AmazonEKSWorkerNodePolicy) which includes permission to describe clusters and subnets.
+
+### Tags
+
+Instances must have the tag `kubernetes.io/cluster/EksClusterName` and value `owned` to successfully join the EKS cluster.
```
Comment (ellistarn) [inline website/content/en/docs/AWS/launch-templates.md:54]: We're EKS Agnostic
```suggestion
Instances must have the tag `kubernetes.io/cluster/$CLUSTER_NAME` and value `owned` to successfully join the EKS cluster. 

```

File: website/content/en/docs/AWS/launch-templates.md [hunk 2]
```diff
@@ -41,6 +41,20 @@ AMI (Amazon Machine Image), is the base image/VM for a launch template.
...
+
+The instance profile must include *at least* the permissions of the default Karpenter node instance profile. See the default role, `KarpenterNodeRole`, in the full example below for more information.
+
+See also, [the managed policy "AmazonEKSWorkerNodePolicy"](https://docs.aws.amazon.com/eks/latest/userguide/security-iam-awsmanpol.html#security-iam-awsmanpol-AmazonEKSWorkerNodePolicy) which includes permission to describe clusters and subnets.
+
+### Tags
+
+Instances must have the tag `kubernetes.io/cluster/EksClusterName` and value `owned` to successfully join the EKS cluster. 
+
+Note that launch templates have two resource tag sections, one for the launch template itself, and another for the instances it creates. Check that the instances are tagged.
```
Comment (ellistarn) [inline website/content/en/docs/AWS/launch-templates.md:56]: We actually override the resource tag w/ fleet overrides, so they don't need to be specified.

**Review by ellistarn — COMMENTED** [review]
Nice! Thank you!

**Comment by geoffcline** [comment]
moving this to WIP status; need to revise desc of security group to say its required;

File: website/content/en/docs/AWS/launch-templates.md (hunk 1 above)
Comment (geoffcline) [reply website/content/en/docs/AWS/launch-templates.md:54]: revised to say this is EKS only;

File: website/content/en/docs/AWS/launch-templates.md (hunk 2 above)
Comment (geoffcline) [reply website/content/en/docs/AWS/launch-templates.md:56]: i'm confused. if karpenter overrides these tags, why does not including them in the LT cause nodes to fail to join?

**Comment by geoffcline** [comment]
updated;

File: website/content/en/docs/AWS/launch-templates.md [hunk 3]
```diff
@@ -41,6 +41,22 @@ AMI (Amazon Machine Image), is the base image/VM for a launch template.
...
+
+The instance profile must include *at least* the permissions of the default Karpenter node instance profile. See the default role, `KarpenterNodeRole`, in the full example below for more information.
+
+See also, [the managed policy "AmazonEKSWorkerNodePolicy"](https://docs.aws.amazon.com/eks/latest/userguide/security-iam-awsmanpol.html#security-iam-awsmanpol-AmazonEKSWorkerNodePolicy) which includes permission to describe clusters and subnets.
+
+### Tags 
+
+ ✳️ Note: this requirement only applies to EKS clusters. 
+
+Instances must have the tag `kubernetes.io/cluster/EksClusterName` and value `owned` to successfully join an EKS cluster.
```
Comment (ellistarn) [inline website/content/en/docs/AWS/launch-templates.md:56]: Mentioned earlier. This has nothing to do with EKS.

File: website/content/en/docs/AWS/launch-templates.md (hunk 2 above)
Comment (ellistarn) [reply website/content/en/docs/AWS/launch-templates.md:58]: This is surprising! It's possible we're not adding all of the right ones? @bwagner5 do you have any idea?

Comment (ellistarn) [reply website/content/en/docs/AWS/launch-templates.md:58]: Implemented here https://github.com/aws/karpenter/commit/9af7af01f94a1a172a38ad404c8158153f488bd7#diff-bc42bdb45b943498089740b1cc573ac30b5cfc994dc76981636daea88ae1d495

File: website/content/en/docs/AWS/launch-templates.md (hunk 3 above)
Comment (geoffcline) [reply website/content/en/docs/AWS/launch-templates.md:56]: This tag isn't required with self managed clusters.

Comment (ellistarn) [reply website/content/en/docs/AWS/launch-templates.md:56]: Did you confirm this one? If so, happy to merge.

File: website/content/en/docs/AWS/launch-templates.md [hunk 4]
```diff
@@ -41,6 +41,22 @@ AMI (Amazon Machine Image), is the base image/VM for a launch template.
 [Review the instructions for importing a VM to AWS.](https://docs.aws.amazon.com/vm-import/latest/userguide/vmimport-image-import.html) Note the AMI id generated by this process, such as,
 `ami-074cce78125f09d61`.
 
+### Instance Profile - IAM
```
Comment (ellistarn) [inline website/content/en/docs/AWS/launch-templates.md:44]: This one is now slightly out of date due to the new CLI Flag

**Comment by geoffcline** [comment]
I'm closing this.

the OG customer issue #432 was resolved by https://github.com/aws/karpenter/commit/9af7af01f94a1a172a38ad404c8158153f488bd7#diff-bc42bdb45b943498089740b1cc573ac30b5cfc994dc76981636daea88ae1d495

the result of my research into this was that 
- EKS requires the label before instances can be admitted
- kOps will technically admit instances, but some cluster node management operations require it
- it's good for everyone to set it

👍

**Pull request closed without merging**